web/src/pages/apd/new/__snapshots__/ApdNew.test.js.snap

```diff
@@ -6,7 +6,7 @@ Array [
     apdname="years[0]"
```

Comment:
I'm realizing this snapshot test I added is probably going to be more annoying than useful. I wanted to have it to sort of alert us when there are changes to the medicaid business areas labels or values. However it seems like any change at all will require updating the snapshot. Curious what others think but im ok if you want to remove it entirely.

Reply:
Clarifying: cut out the snapshots for ApdNew? or all of them?